task/02-numbers-tasks.js

```diff
@@ -89,7 +89,7 @@ function getDistanceBetweenPoints(x1, y1, x2, y2) {
  *   5*x = 0         => 0
  */
 function getLinearEquationRoot(a, b) {
-    throw new Error('Not implemented');
+    return (b === 0) ? 0 : -b / a;
```

Comment:
> Actually, we don't need to check b === 0, because -b / a equals zero if (b ===0)
Hello. If i take away this check (b === 0), the result of the function will be -0 (minus zero). In this case the test with parameters a=5 and b = 0 fails.

Reply:
> > Actually, we don't need to check b === 0, because -b / a equals zero if (b ===0)
> > Hello. If i remove this check (b === 0), the result of the function will be -0 (minus zero). In this case the test with parameters a=5 and b = 0 fails.

But it shouldn't fail, because as we can see on screenshot (please, try it in your chrome console  0 === -0):
![Selection_036](https://user-images.githubusercontent.com/26328278/90752017-88448e00-e2df-11ea-85d8-8501c6ec7575.png)

Reply:
But we have successful tracis build for such solution here:
#928

https://travis-ci.org/github/AleshaKolesnikov/js-assignments/builds/719550067
Please, try to find differences.
May be node versions, travis.yml